portfolio_ahardman_stylesheet.css

```diff
@@ -0,0 +1,261 @@
+/*   default settings   */
+*{
+box-sizing: border-box;
+font-family:Segoe UI, Helvetica, Verdana; 
+}
+body{                 	/* removes default padding and margin around the ouside of the entire webpage */
+	font-size:62.5%;
+	margin:0;
+	padding:0;
+}
+@-ms-viewport{
+  width: device-width;
+}
+a{text-decoration:none;
+}
+input:focus{
+	decoration:none;
+	border:none;
+	outline:none;
+}
+/* navbar settings */
+.navbar{
+	position:sticky;
+	display:flex;
+	justify-content:space-between;
+	align-items:center;
+	top:0px;
+	width:100%;
+	height:10vh;
+	border:1px solid black;
+}
+.navbar .left.container{
+	height:100%;
+	width:60%;
+	border:1px solid rgb(255,255,0);
+}
+.navbar .link.container{
+	width:40%;
+	height:100%;
+	display:flex;
+	justify-content:space-around;
+	align-items:center;
+}
+.navbar  .link.container a{
+	color:rgb(170,170,170);
+	font-size:1rem;
+	transition-duration:0.8s;
+}
+.navbar  .link.container a:visited{
+	color:inherit;
+	text-decoration:none;
+}
+.navbar  .link.container a:hover{
+	color:rgb(255,255,255);
+	transition-duration:0.3s;
+}
+
+/* main section settings*/
+main{
+	display:flex;
+	justify-content:center;
+	align-items:center;
+	width:100%;
+	height:90vh;
+	border:1px solid black;
+}
+main .container{
+	display:flex;
+	justify-content:center;
+	flex-direction:column;
+	height:50%;
+	width:70%;
+	border:1px solid black;
+}
+main h1{
+	font-size:5rem;
+	margin:0px;
+	padding:0px;
+	border:0px;
+}
+main h3{
+	text-align:left;
+	font-size:2rem;
+	margin:30px 0px 0px 0px;
+	padding:0px;
+	border:0px;
+}
+/* portfolio section */
+.portfolio{
+	width:100%;
+	height:700px;
+	display:flex;
+	justify-content:center;
+	align-items:center;
+	border:1px solid red;
+}
+.portfolio .container{
+	width:70%;
+	height:500px;
+	background-color:blue;
+}
+.portfolio h2{
```

Comment:
Because you have defined the height of the box and the font-size, unless the font-size and height are equal the text will not appear center in the y-axis. This applies to all your headers in your different sections.